### PR TITLE
DRAFT: Introduce one-init

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This is draft work to introduce one-init.

for #9450
after #9369

ONE-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

File: compiler/one-cmds/tests/one-init-002.test [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```
Comment (hyunsik-yoon) [inline compiler/one-cmds/tests/one-init-002.test:1]: I have to figure out how to manually run this test. 
Any help could save my time a lot!

Comment (seanshpark) [reply compiler/one-cmds/tests/one-init-002.test:1]: (1) make a local setup with
```
NNAS_BUILD_PREFIX=build/release \
NNCC_WORKSPACE=build/release \
BUILD_TYPE=Release \
./nnas create-package --preset 20220323 --prefix build/install
```
(2) prepare test materials
```
cd build/install/bin/test
./prepare_test_materials.sh
```

(3) run test
```
bash ./one-init-002.test
```

Comment (yunjayh) [reply compiler/one-cmds/tests/one-init-002.test:1]: In my case for (3), I commented out the tests except what I want in `build/install/test/runtestall.sh` and ran the script.

Comment (hyunsik-yoon) [reply compiler/one-cmds/tests/one-init-002.test:1]: @seanshpark,
For (2),
I couldn't find `build/install/bin/test/prepare_test_materials.sh`.
Should I run `build/install/test/prepare_test_materials.sh`?

```
$ find build  -name prepare_test_materials.sh
build/install/test/onnx-operations/prepare_test_materials.sh
build/install/test/prepare_test_materials.sh
```

**Comment by hyunsik-yoon** [comment]
#9514 is the next version.

**Pull request closed without merging**